// The following code must be successfully parsed and not result in an error
package main;

var err string = "";

func div(numerator, denominator int) int {
    if denominator == 0 {
        err = "can't divide by zero";
        return 0;
    };

    return numerator / denominator;
};

func panic(a string){};

func main() {
    result := div(3, 0);
    if err != "" {
        panic(err);
    };
};
